### DISCUSSION TITLE
Search whole file on select if current selection is a single char

### DISCUSSION THREAD

**Discussion by erf** [helix-editor/helix#10273] [General]
I would like to select / search the whole file without making a selection and avoid having the cursor move to the bottom of the file.

This might be an exception to the general rule, as there is always a selection, but If only a single char is selected, could we search the whole file then, as searching a single character does not make much sense anyways?

**Comment by archseer** [comment]
What do you mean by searching? `/` already searches the whole file

**Comment by erf** [comment]
I mean select `s`, the whole file, if only a single char is selected.
